server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -642,7 +643,7 @@ private void getMonitorLock() throws KeeperException, InterruptedException {
       UtilWaitThread.sleep(getContext().getConfiguration().getTimeInMillis(Property.MONITOR_LOCK_CHECK_INTERVAL));
     }
 
-    log.info("Got Monitor lock.");
+    log.info("Acquired Monitor Lock " + monitorLock.getLockPath());
```

Comment:
Parameterized messages eliminate unnecessary object creation and we are moving to use that throughout the code base.  Without the parameter format, monitor.getLockPath() is always called, even if the log message will not be written. When passed as a parameter, the evaluation is deferred until the log statement has been determined that the value is going to be used.